### PR TITLE
Hfvqe tutorials

### PR DESCRIPTION
**Pull request by ncrubin**

### CODE REVIEW

File: docs/hfvqe/Making MolecularData.ipynb [hunk 1]
```diff
@@ -0,0 +1,566 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Making Molecule Files for HFVQE\n",
```
Comment (mpharrigan) [inline docs/hfvqe/Making MolecularData.ipynb:7]: can you name this file in `lower_snake_case` with no spaces

File: docs/hfvqe/Making MolecularData.ipynb [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
...NeBrRZPYImIiu1+jovMmiS0ieqXGFhHtYjw6WncQpSSxRcREB7YtGmBJbBHRK9M9IqJNDDg1tohoFTs1tohon0EfPJD7OKwr6WHgv2e4bQWwpw/hFJV4Zta0mBZyPC+2/YIyBUi6mk7MReyxva7M8+ZDXxNbEZJGnm3jun5LPDNrWkyJJ7JtUUS0ThJbRLROExPbproDmCTxzKxpMSWeBa5xfWwREWU1scYWEVFKEltEtE5jEpukdZLulrRd0oxvoelDPGsk/VTSnZLukHRu3TEBSBqSdIukHzYgludJ2iLpV5LukvS6muP5WPd3dbukyyUdXEMMl0jaLen2cZ8tl3StpHu6X5f1O66FphGJTdIQcBHwNuB44AxJx9cbFfuBT9g+HjgZ+JsGxARwLnBX3UF0fQ242vYrgD+mxrgkrQI+Quf1bCcAQ8DpNYTybWDyhNXzgOtsHwdc172OedSIxAacBGy3fa/tfcAVwPo6A7K9y/bN3fPf0/mfdlWdMUlaDbwDKPVqsopiOQJ4I/BNANv7bP9vrUF1lgg+V9Ji4BDggX4HYPtnwN5JH68HNnfPNwPv6mdMC1FTEtsqYMe4653UnETGk3QMcCJwQ82hXAh8CmjCCuVjgYeBb3WbxhdLWlpXMLbvB74E3AfsAh61/aO64plkpe1d3fMHgZV1BrMQNCWxNZakQ4HvAx+1/bsa43gnsNv2TXXFMMli4NXA122fCDxOjU2sbr/VejoJ90XAUkln1hXPdNyZX5U5VvOsKYntfmDNuOvV3c9qJek5dJLaZbavrDmcU4DTJP0Xnab6WyRdWmM8O4Gdtg/UYrfQSXR1WQv81vbDtp8GrgReX2M84z0k6SiA7tfdNcfTek1JbDcCx0k6VtISOp2+W+sMSJLo9B/dZfsrdcYCYPvTtlfbPobOz+cntmurkdh+ENgh6eXdj04F7qwrHjpN0JMlHdL93Z1KcwZZtgIbuucbgB/UGMuC0Ij92Gzvl3QOcA2d0axLbN9Rc1inAO8Hfinp1u5nf2d7W30hNc6Hgcu6f4zuBT5QVyC2b5C0BbiZzoj2LdSwlEnS5cCbgRWSdgKfBS4A/lnS2XS27XpPv+NaaLKkKiJapylN0YiIyiSxRUTrJLFFROsksUVE6ySxRUTrJLFFROsksUVE6/wfchBC52bJccYAAAAASUVORK5CYII=\n",
```
Comment (mpharrigan) [inline docs/hfvqe/Making MolecularData.ipynb:516]: Can we commit this without outputs? At least to get git to realize this is a rename and not a wholesale deletion and addition

**Review by mpharrigan — CHANGES_REQUESTED** [review]

**Comment by mpharrigan** [comment]
There's also the "odd qubit" example notebook still in the hfvqe module. What do you want to do with that?

**Comment by mpharrigan** [comment]
Discussed offline: the hfvqe readme should be ported to docs/hfvqe/index.md to serve as a "landing page"

**Comment by ncrubin** [comment]
@mpharrigan I see master already had the odd qubit tutorial. I was going to leave it in hfvqe since it doesn't add much to the example.  I will open another PR to remove.

**Comment by ncrubin** [comment]
@mpharrigan let me know if I need to make any other structural changes.

**Review by mpharrigan — APPROVED** [review]

**Pull request merged**